cts/cts-exec.in

```diff
@@ -85,18 +86,23 @@ class ExecTest(Test):
 
         self.tls = kwargs.get("tls", False)
 
+        # If we are going to run the stonith resource tests, we will need to
+        # launch and track Corosync and pacemaker-fenced.
+        self._corosync = None
+        self._fencer = None
+
         if self.tls:
             self._daemon_location = "pacemaker-remoted"
         else:
             self._daemon_location = "pacemaker-execd"
+            if self.name.count("stonith") != 0:
```

Comment:
It might be more pythonic (ugh) to say `if "stonith" in self.name:`

Reply:
Updated commit